### ISSUE TITLE
Extend `TorConnectionFields` Struct with Additional Fields

### ISSUE THREAD

**Issue by msk** [petabi/review-database#49]
**Issue Description:**

The current `TorConnectionFields` struct in our Tor traffic monitoring module encapsulates a basic set of fields:

```rust
pub(super) struct TorConnectionFields {
    pub source: String,
    pub src_addr: IpAddr,
    pub src_port: u16,
    pub dst_addr: IpAddr,
    pub dst_port: u16,
    pub proto: u8,
}
```

To ensure a more comprehensive view of Tor connections, we need to enhance this struct to incorporate additional fields. The updated `TorConnectionFields` struct should look as follows:

```rust
pub(super) struct TorConnectionFields {
    #[serde(with = "ts_nanoseconds")]
    pub session_end_time: DateTime<Utc>,
    pub source: String,
    pub src_addr: IpAddr,
    pub src_port: u16,
    pub dst_addr: IpAddr,
    pub dst_port: u16,
    pub proto: u8,
    pub method: String,
    pub host: String,
    pub uri: String,
    pub referrer: String,
    pub version: String,
    pub user_agent: String,
    pub request_len: usize,
    pub response_len: usize,
    pub status_code: u16,
    pub status_msg: String,
    pub username: String,
    pub password: String,
    pub cookie: String,
    pub content_encoding: String,
    pub content_type: String,
    pub cache_control: String,
}
```

These additional fields will provide richer information about each Tor connection, including session end time, HTTP method, host, URI, referrer, HTTP version, user agent, request length, response length, status code, status message, username, password, cookie, content encoding, content type, and cache control.

**Tasks:**

- Extend the `TorConnectionFields` struct with the new fields.
- Update the function(s) that instantiate and use `TorConnectionFields` to accommodate the new fields.
- Add a database migration function to handle the new fields in existing data, filling the new fields with default values.
- Create unit tests to verify the correct functionality of the migration.

**Comment by kimhanbeom** [comment]
The contents of this issue have already been merged (https://github.com/petabi/review-database/pull/51) and should be closed.

**Comment by msk** [comment]
Resolved in #51.
